code/game/machinery/droneDispenser.dm

```diff
@@ -174,7 +174,7 @@
 				use_power(power_used)
 
 			var/atom/A = new dispense_type(loc)
-			A.admin_spawned = admin_spawned
+			A.flags_1 |= (flags_1 | ADMIN_SPAWNED_1)
```

Comment:
Incorrect logic here, should be more like
```dm
A.flags_1 = (A.flags_1 & ~ADMIN_SPAWNED_1) | (flags_1 & ADMIN_SPAWNED_1)
```
or so. Ditto in `drones_as_items.dm`, `hivelord.dm`, and `spawner.dm`.

Reply:
This isn't intended to clear the admin spawned flag if it's set on the new atom, though it is guaranteed to not be set on any new atom. "admin spawned" is, as far as i understand it, intended to be a viral property that taints an atom and any atom it creates.